tests/std/tests/P0896R4_views_elements/test.cpp

```diff
@@ -394,6 +394,27 @@ constexpr void instantiation_test() {
 #endif // TEST_EVERYTHING
 }
 
+// GH-3014 "<ranges>: list-initialization is misused"
+void test_gh_3014() { // COMPILE-ONLY
+    using P = pair<int, int>;

```

Comment:
This shadows an identical typedef at file scope:
https://github.com/microsoft/STL/blob/e7f52446c6f41fddbed31904f594a1f711f85601/tests/std/tests/P0896R4_views_elements/test.cpp#L15